charts/trino/values.yaml

```diff
@@ -55,9 +55,21 @@ server:
     authenticationType: ""
     query:
       maxMemory: "4GB"
-  exchangeManager:
-    name: "filesystem"
-    baseDir: "/tmp/trino-local-file-system-exchange-manager"
+  exchangeManager: {}
+  # exchangeManager -- Mandatory [exchange manager
+  # configuration](https://trino.io/docs/current/admin/fault-tolerant-execution.html#id1).
+  # To configure additional exchange manager properties,
+  # add them in `additionalExchangeManagerProperties`.
+  # @raw
+  # Set the name of the spooling storage destination:
```

Comment:
It would be better to have one complete example.

Reply:
I've added the `additionalExchangeManagerProperties` to the example, but I'm thinking maybe it's more confusing than helpful to include them there, whereas they don't belong to the `.Values.server` hierarchy..

Reply:
That's ok, doesn't need to be perfect, any docs are better than no docs.

Reply:
You went back to separate yaml sections. I think it would be better to have one complete example. You could just copy the contents of `test-exchange-manager-values.yaml` here, even with the `additionalConfigProperties`. Some users might wonder why the exchange settings are not effective and forget to change the retry policy.

Reply:
Agree, changed.